specific/cmo_project/data/cmo_project_sequence.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+    <data noupdate="1">
+        <record id="seq_type_cmo_project" model="ir.sequence.type">
+            <field name="name">CMO Project</field>
+            <field name="code">cmo.project</field>
+        </record>
+        <record id="seq_cmo_project" model="ir.sequence">
+            <field name="name">CMO Project (CMO)</field>
+            <field name="code">cmo.project</field>
+            <field name="prefix">CM%(y)s-</field>
```

Comment:
Will the project number be reset every beginning of year?
According to what you did here -> CM%(y)s the number will run as CM20160001 ... CM**2016**0012, then, CM**2017**0013
Sure? (if customer is ok, I am fine)
**Note:** normally, I will see that customer want to reset number every beginning of year, i..e, CM**2017**0001